doc/whats_new/v0.24.rst

```diff
@@ -52,6 +52,10 @@ Changelog
   unless data is sparse.
   :pr:`17396` by :user:`Jiaxiang <fujiaxiang>`.
 
+- |Enhancement| :func:`datasets.fetch_20newsgroups_vectorized` now supports
+  heterogeneous data using pandas by setting `as_frame=True`.
```

Comment:
I'm not sure I understand the comment, the feature are always numeric, right? That's why it's vectorized? The target is strings but I wouldn't consider that heterogeneous as X is homogeneous.

Reply:
oh, I just blindly copy pasted the changelog from the california housing dataset without thinking about the meaning of it. I'll fix it.